components/Footer.tsx

```diff
@@ -0,0 +1,139 @@
+// Footer.js
+import React from "react";
+import "../styles/Footer.css";
+import { footerLogo } from "../components/assets";
+
+const Footer = () => {
+  return (
+    <footer
+      className="flex-col align-center bg-neutral-900 w-screen"
+      id="footer"
+    >
+      <div className="w-screen max-w-screen-lg mx-auto grid grid-cols-1 sm:grid-cols-3 md:grid-cols-5 justify-items-center pt-16 px-16">
+        <div
+          className="col-auto max-w-2xl flex flex-col sm:flex-row sm:col-span-3 md:col-span-2 md:flex-col lg:flex-row items-center"
+          style={{ maxWidth: "25rem" }}
+        >
+          <img className="h-60 m-auto" src={footerLogo} alt="Footer Logo" />
+          <div style={{ width: "20rem" }}>
+            <p
+              className="text-neutral-300 text-center sm:text-left md:text-center lg:text-left"
+              id="desc"
+            >
+              Advisory is a fully youth-led non-profit dedicated to empowering
+              young Singaporeans to make informed decisions about their futures.
+            </p>
+          </div>
+        </div>
```

Comment:
Could you shift the privacy banner to the footer? I leave the positioning up to you (Angela will review that later), but I would also prefer the footer to be smaller in height if possible. Try to match the sizing of the logo with the beta website, I think they seem a bit different:

![image](https://github.com/AdvisorySG/mentorship-page/assets/2070423/e157552e-e711-445d-bd9a-6955613d82b4)
(Mentorship)

vs

![image](https://github.com/AdvisorySG/mentorship-page/assets/2070423/b56b2513-3b2c-45cf-b1f5-e80c049eb021)
(Ghost)

Reply:
Might be because `rem` is different. In that case, see if you can standardise it, it's not a big issue if the existing font size changes.